.github/workflows/clp-core-build.yaml

```
@@ -2,27 +2,14 @@ name: "clp-core-build"
   push:
-    paths:
-      - ".github/actions/clp-core-build-containers/action.yaml"
-      - ".github/actions/run-on-image/action.yaml"
-      - ".github/workflows/clp-core-build.yaml"
-      - ".gitmodules"
-      - "components/core/**"
-      - "taskfile.yaml"
-      - "taskfiles/**"
-      - "tools/scripts/deps-download/**"
-      - "!components/core/tools/scripts/lib_install/macos/**"
+    paths: *monitored_paths
```

actionlint (1.7.7)

"paths" section must be sequence node but got alias node with "" tag

(syntax-check)